config/SavedVariables.lua

```diff
@@ -156,6 +157,10 @@ ns.EventHandler:Register('ADDON_LOADED', function(...)
 			QuickQuestBlacklistDB = nil
 		end
 
+		if not ns.db.profile.blocklist.title then
+			ns.db.profile.blocklist.title = {}
+		end
```

Comment:
AceDB should be able to merge this by itself, let me know if it doesn't.

Reply:
cool I wasn't sure what ace's behavior was 👍